Sources/SwiftyMarkdown/SwiftyMarkdown+iOS.swift

```diff
@@ -129,12 +129,18 @@ extension SwiftyMarkdown {
 			font = UIFont.preferredFont(forTextStyle: textStyle)
 		}
 		
-		if globalItalic, let italicDescriptor = font.fontDescriptor.withSymbolicTraits(.traitItalic) {
-			font = UIFont(descriptor: italicDescriptor, size: fontSize ?? 0)
-		}
-		if globalBold, let boldDescriptor = font.fontDescriptor.withSymbolicTraits(.traitBold) {
-			font = UIFont(descriptor: boldDescriptor, size: fontSize ?? 0)
-		}
+        var traits: UIFontDescriptor.SymbolicTraits = []
+        if globalItalic {
+            traits.insert(.traitItalic)
+        }
+        if globalBold {
+            traits.insert(.traitBold)
+        }
```

Comment:
In theory this would also allow text to be both bold and italic, which wouldn't work before, however it seems like other bugs in the parsing prevent that.